src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.elevator;
+
+import com.ctre.phoenix6.controls.ControlRequest;
+import com.ctre.phoenix6.signals.ControlModeValue;
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+
+import static frc.robot.subsystems.elevator.ElevatorConstants.*;
+
+
+public class Elevator extends SubsystemBase {
```

Comment:
You don't call `processInputs()`.

Reply:
**Insert Jamaican accent** yeah mon.

Reply:
2921fdc3f5ddf12009a0bf12e0b8eab343409f09

Reply:
This is not a relevant commit. Also, you don't need to create `toLog()` and `fromLog()`, `@AutoLog` does it by itself.

Reply:
Fixed that, but I cant find the commit